models.py

```diff
@@ -584,8 +587,22 @@ def forward(self, signal: shaping.BT, mask: shaping.BT = None, **kwargs) -> shap
 
 		power_spectrum = real_squared + imag_squared
 		log_mel_features = self.mel(power_spectrum).log()
+
+		if self.check_features_divisibility_by_32:
```

Comment:
please move this check into JasperNet's forward, let's start with the check there